tiflash/use-tiflash.md

```diff
@@ -217,15 +217,24 @@ You can configure this parameter in one of the following ways:
 
 ## Supported push-down calculations
 
-TiFlash supports predicate, aggregate push-down calculations, and table joins. Push-down calculations can help TiDB perform distributed acceleration. Currently, `Full Outer Join` and `DISTINCT COUNT` are not the supported calculation types, which will be optimized in later versions.
+TiFlash supports the push-down of the following operators:
 
-You can enable the push-down of `join` using the following session variable (`Full Outer Join` is currently not supported):
+* TableScan: Reads data from tables.
+* Selection: Filters data.
+* HashAgg: Performs data aggregation based on the [Hash Aggregation](/explain-aggregation.md#hash-aggregation) algorithm.
+* StreamAgg: Performs data aggregation based on the [Stream Aggregation](/explain-aggregation.md#stream-aggregation) algorithm. SteamAgg only supports the column without the `GROUP BY` condition.
```

Comment:
SteamAgg only supports the column without the => SteamAgg only supports the aggregation without the